articles/how-to-choose-levels-of-logging/index.md

```diff
@@ -0,0 +1,65 @@
+
+#### ERROR
+
+Unlike FATAL logging level, your application is not aborting. Instead, there is just an inability to access a service or a file. This ERROR shows a failure of something important in your application. This log level is used when a severe issue is stopping functions within the application from operating efficiently. Most of the times, the application will continue to run, but eventually, it will need to be addressed.
+
+#### WARN
+
+The WARN log level is used when you have detected an unexpected application problem. And you are not quite sure whether the problem will recur or remain. You may not notice any harm to your application at this point. This issue is usually a situation that stops specific processes from running. But it does not mean that the application has been harmed. In fact, the code continues to work as usual. You should eventually check these warnings just in case the problem recurs.
+
+#### INFO
+
+INFO messages are like the normal behavior of applications. They state what happened. For example, if a particular service stopped or started or you added something to the database. These entries are nothing to worry about during usual operations. The information logged using the INFO log is usually informative, and it does not necessarily require you to follow up on it.
+
+#### DEBUG
+
+With DEBUG, you are giving diagnostic information in a detailed manner. It is verbose and has more information than you would need when using the application. DEBUG logging level is used to fetch information needed to diagnose, troubleshoot, or test an application. This ensures the smooth running of an application.
+
+#### TRACE
+
+TRACE log level captures all the details about the behavior of the application. It is mostly diagnostic and is more granular and finer than DEBUG log level. This log level is used in situations where you need to see what happens in your application or what happens in the third-party libraries used. You can use the TRACE log level to query parameters in the code or interpret the algorithm's steps.
+
+#### ALL
+
+This log level logs any logging levels that are defined. It logs everything and includes custom logging levels as well. It is the combination of all other logging levels.
+
+#### OFF
+
+This log level does not log anything. This OFF level is used for turning off logging and is the greatest possible rank. With this log level, nothing gets logged at all.
+
+### Do log levels work on their own?
+
+Logging levels do not work well on their own. They mostly serve information purposes in the log message. They provide information on how crucial an event is. You can limit the information written to the log level by combining the log level with a logging framework.
+
+Setting a specific log level in the logging framework ensures less important logging levels are ignored. For example, if the logging framework has the root log level as WARN, you only get events with FATAL level, ERROR level, and WARN level.
+
```

Comment:
Looks like the log levels are ordered. Consider turning them into an ordered list. Also, state how they are ordered. ie. according to what criteria and in what order.